File: Coursework/Level_5/fare_calculator.py
calculate_fare = lambda distance: 200 + (distance * 50)

distance = float(input("Enter distance (km): "))
total_fare = calculate_fare(distance)

print("\nBase Fare: 200 KES")
print(f"Distance: {distance} km")
print("Cost per km: 50 KES")
print(f"Total Fare: {total_fare} KES")

File: Coursework/Level_5/FareCalculator.java
import java.util.Scanner;

class Taxi {
    double baseFare = 200;
    double costPerKm = 50;
    
    double calculateFare(double distance) {
        return baseFare + (distance * costPerKm);
    }
}

public class FareCalculator {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        Taxi taxi = new Taxi();
        
        System.out.print("Enter distance (km): ");
        double distance = sc.nextDouble();
        
        double totalFare = taxi.calculateFare(distance);
        
        System.out.println("\nBase Fare: 200 KES");
        System.out.println("Distance: " + distance + " km");
        System.out.println("Cost per km: 50 KES");
        System.out.println("Total Fare: " + totalFare + " KES");
        
        sc.close();
    }
}
